jujugui/static/gui/src/app/app.js

```diff
@@ -1828,21 +1842,27 @@ YUI.add('juju-gui', function(Y) {
     @method destructor
     */
     destructor: function() {
+      if (this.dbChangedTimer) {
```

Comment:
Yay thanks!
This definitely deserves a code comment as to why this is here and to never remove it :smile: